src/components/CurriculumComponents/OakComponentsKitchen/Terms/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { OakLink, OakP } from "@oaknational/oak-components";
+import { ComponentProps } from "react";
```

Comment:
I'm so sorry but I think this component also exists as a `CopyrightNotice` in TeacherComponents 😢

Reply:
It's actually more interesting than that, there already two in the codebase

 - https://github.com/oaknational/Oak-Web-Application/blob/main/src/components/PupilComponents/CopyrightNotice/CopyrightNotice.tsx
 - https://github.com/oaknational/Oak-Web-Application/blob/main/src/components/TeacherComponents/CopyrightNotice/CopyrightNotice.tsx
 
This is intended to be upstreamed to `oak-components`... so one more... 😬

Seriously lets make this one the priority to upstream